packages/p2p/src/components/orders/chat/chat-footer.jsx

```diff
@@ -15,18 +15,8 @@ const ChatFooter = observer(() => {
     const text_input_ref = React.useRef(null);
     const [character_count, setCharacterCount] = React.useState(0);
 
-    const updateTextAreaBounds = () => {
-        const el_target = text_input_ref.current;
-
-        if (el_target) {
-            el_target.setAttribute('style', 'height: auto;');
-            el_target.setAttribute('style', `height: ${el_target.scrollHeight}px;`);
-        }
-    };
-
```

Comment:
No need to update the styling here, this is what caused the footer to change height in firefox. Better to handle all styles in scss file itself and not through javascript